src/DotVVM.Framework/ResourceManagement/ResourceManager.cs

```diff
@@ -131,6 +141,16 @@ public void AddRequiredStylesheetFile(string name, string url, params string[] d
             });
         }
 
+        /// <summary>
+        /// Adds the required inline stylesheet.
+        /// </summary>
+        public void AddRequiredInlineStylesheet(string name, string code, params string[] dependentResourceNames)
```

Comment:
I'm not sure how would I name it, since it's so similar to `AddStartupScript` which has no Required in name... but AddStartupStyle is obviously also terrible :D

Reply:
I have no idea how to name this method so it would be descriptive and similar to `AddStartupScript` .